### ISSUE TITLE
No way to change keybindings of action Menu

### ISSUE THREAD

**Issue by ujvaljoshi** [atg/chocolat-public#1774]
OS: 10.11.3
Chocolat App Version: 3.2.4

I cant find option to change key binding of these options in action menu.

![bug](https://cloud.githubusercontent.com/assets/870044/13388023/834504a8-dee2-11e5-9929-f5081c174653.png)

**Comment by atg** [comment]
Yeah the actions menu is blacklisted. I guess I could give the top items a different `tag` and whitelist them

**Comment by ujvaljoshi** [comment]
Issue i am facing is with cmd + R. Browser reloading is also same. When u press cmd + r while switching from chocolat to browser this command opens terminal.

**Comment by atg** [comment]
for that particular issue, changing the key equiv wouldn't help because actions -> run is what does the refreshing

**Comment by ujvaljoshi** [comment]
I want to change CMD + R to anything. Because pressing CMD + R in HTML document open this kind of terminal window.

![screenshot 2016-03-02 15 46 36](https://cloud.githubusercontent.com/assets/870044/13457393/05bd2134-e08e-11e5-816d-479effb3acb4.png)
